cogs/games.py

```diff
@@ -33,10 +36,436 @@
+
+    async def prompt(self, ctx) -> dict:
+        await self.new_game()
+        await self.start(ctx, wait=True)
+        return {P.player_id: P.balance for P in self.players}
+
+
+class Blackjack_players(menus.Menu):
+    def __init__(self, author, author_money, cost, **kwargs) -> None:
+        super().__init__(**kwargs)
+        self.players = [author]
+        self.money_dict = {author.id: author_money}
+        self.cost = cost
+
+    def reaction_check(self, payload) -> bool:
+        return payload.message_id == self.message.id and payload.user_id != self.bot.user.id
+
+    async def send_initial_message(self, ctx, channel) -> Message:
+        self.time = 120
+        self.updater.start()
+        return await ctx.send(embed=self.get_embed())
+
+    @tasks.loop(seconds=5)
+    async def updater(self) -> None:
+        self.time -= 5
+        await self.message.edit(embed=self.get_embed())
+        if self.time == 0:
+            self.stop()
+
+    @updater.before_loop
+    async def waiter(self) -> None:
+        await asyncio.sleep(5)
+
+    def get_embed(self) -> Embed:
+        r = "\n -"
+        ini = "Check the command's help for the rules. React with :white_check_mark: to join, :track_next: to begin the game\n\n"
+        return Embed(
+            title=f"Come play blackjack ! Initial bet is {self.cost} GP ({self.time} seconds left)",
+            description=f"{ini}Current players :\n -{r.join([player.mention for player in self.players])}",
+        )
+
+    @menus.button("\U00002705")
+    async def adder(self, payload) -> None:
+        member = self.ctx.guild.get_member(payload.user_id)
+        if member in self.players:
+            self.players.remove(member)
+            del self.money_dict[payload.user_id]
+        else:
+            self.money_dict[payload.user_id] = 100
+            self.players.append(member)
+
+    @menus.button("\U000023ed\N{variation selector-16}")
+    async def skipper(self, payload) -> None:
+        self.stop()
+
+    async def prompt(self, ctx) -> tuple:
+        await self.start(ctx, wait=True)
+        return self.players, self.money_dict
+
+    def stop(self) -> None:
+        self.updater.stop()
+        super().stop()
+
+
 class Games(Cog):
     def __init__(self, bot: Bot) -> None:
         self.bot = bot
 
+    @command(ignore_extra=True)
+    async def blackjack(self, ctx: Context, cost: int = 5) -> None:
+        """Please see the detailed help
+        Rules : if it's your turn, press the button corresponding to the column in which you wana place the card.
+        If you want to split (play on one more column, up to a max of 3, press :regional_indicator_3:). If you want to stop, press :x:.
+        To win, you must score more than the dealer, but no more than 21 (each card's value is its pip value, except faces, which are worth 10 \
+        points, and the Ace, which is worth either 1 or 11).
+        An Ace plus a face is called a blackjack, and beats a 21"""
```

Comment:
Docstring formatting adjustment.
```suggestion
        """
        Rules: if it's your turn, press the button corresponding to the column in which you want to place the card.
        If you want to split (play on one more column, up to a max of 3, press :regional_indicator_3:).  If you want to stop, press :x:.
        To win, you must score more than the dealer, but no more than 21 (each card's value is its pip value, 
        except for faces, which are worth 10 points, and the Ace, which is worth either 1 or 11).
        An Ace plus a face is called a blackjack and beats a 21
        """
```